### PR TITLE
tweak name

### PR DESCRIPTION
**Pull request by geekplux**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const defaults = {
   d3Module: require('d3'), // to allow use of d3.v4
   selector: '',  // selects base D3 Element
   container: '', // markup inserted in body
-  svgStyles: ''  // embedded svg stylesheets
+  styles: ''  // embedded svg stylesheets
```
Comment (bradoyler) [inline index.js]: @geekplux this is a breaking API change. You should create an alias for this prop first.

**Comment by bradoyler** [comment]
@geekplux Please don't merge unless all tests pass.

**Comment by geekplux** [comment]
I remember 👍

**Comment by geekplux** [comment]
I would send you a review request when next PR, thanks

**Pull request merged**